gildniy/WayFarer
# #167254073 Ft view a trip

#### What does this PR do?
View an individual trip
#### Description of Task to be completed?
The trip with all its details are shown to the user in order to let him/her know which one may be of his/her interest
#### How should this be manually tested?
By cloning the site or navigating to the GitHub page: [https://gildniy.github.io/WayFarer/UI](url) , after clicking on the buttons 'TEST AS USER' or 'TEST AS ADMIN', from the dashboard, on the list of all trip, a green button labeled 'OPEN' is present, and when the user clicks on it, he/she should see a screen showing an image of a bus-like graph with a table showing all information about the current trip:
- DEPARTURE
- DESTINATION
- DATE
- TIME 
- BUS PLATE NO.
- BACK TO THE LIST button
#### What are the relevant pivotal tracker stories?
#167254073
